src/index.js

```diff
@@ -1,5 +1,15 @@
 import ReactDOM from 'react-dom';
 import App from './App/App.js';
 import React from 'react';
+import { createStore } from 'redux';
```

Comment:
for team convention, `createStore` should be called inside `redux/store.js`. reach out to me for more clarif..